.github/stale.yml

```diff
@@ -1,5 +1,5 @@
 # Number of days of inactivity before an issue becomes stale
-daysUntilStale: 30
```

Comment:
ohh, I think 30 days is quite unpopular already. I suppose people think bug=closed means it's not a bug, rather than just we're just not working on a specific bug.

Reply:
I think that's why the `wontfix` tag though

Reply:
Right, no complaint, just sharing thought.